q2_deblur/_denoise.py

```diff
@@ -147,13 +147,12 @@ def _denoise_helper(
                                              pd.DataFrame):
     _check_inputs(**locals())
     with tempfile.TemporaryDirectory() as tmp:
+        df = demultiplexed_seqs.manifest.view(pd.DataFrame)
+        for id in df.index:
```

Comment:
How about you use pandas built-ins for this?

```python
 if any(df.index.str.contains('_')):
    raise ...
```

Reply:
This doesn't keep track of which ID has an underscore in it, do we care about that?

Reply:
> do we care about that

Yep! I am sending you on a quest to learn more about pandas - have fun! :panda_face: